src/badguy/badguy.cpp

```diff
@@ -1128,4 +1129,14 @@ BadGuy::add_wind_velocity(const Vector& velocity, const Vector& end_speed)
     m_physic.set_velocity_y(std::max(m_physic.get_velocity_y() + velocity.y, end_speed.y));
 }
 
+
+void
+BadGuy::register_class(ssq::VM& vm)
+{
+  ssq::Class cls = vm.addAbstractClass<BadGuy>("BadGuy");
```

Comment:
BadGuy doesnt inherit anything meaning that functions like `set_pos()` cant be used on it (rip scriptable granito) and some functions are missing that were there before such as `set_sprite()`.

Reply:
Oh, must be an oversight! Will fix.

Reply:
Yippee